setup.py

```diff
@@ -24,5 +25,8 @@
     author_email='oscm@edx.org',
     license='Apache',
     packages=find_packages(exclude=['*.tests']),
-    install_requires=['slumber >= 0.7.1, < 1.0', 'PyJWT >= 1.1.0, < 2.0'],
+    install_requires=[
+        'slumber>=0.7.1',
+        'PyJWT>= 1.1.0'
```

Comment:
I can see how that would be a problem if the min and max versions were both on the same major version. However, in this case the min and max versions are on different major versions, which seems reasonable.

Put differently, it doesn't make sense to limit this package to PyJWT>=1.1.0,<1.2.0. But PyJWT>=1.1.0,<2.0 makes sense; we don't guarantee that this package will work with PyJWT >=2.0.